parcels/particlefile.py

```diff
@@ -140,6 +147,9 @@ def write(self, pset, time, sync=True):
                     self.z[inds, self.idx] = np.array([p.depth for p in pset])
                     for var in self.user_vars:
                         getattr(self, var)[inds, self.idx] = np.array([getattr(p, var) for p in pset])
+                    if self.idx == 0:
+                        for var in self.user_vars_once:
+                            getattr(self, var)[inds] = np.array([getattr(p, var) for p in pset])
```

Comment:
The same type of code snippet also needs to be added to the ` elif self.type is 'indexed':` below